### PR TITLE
Added methods spy and stub to mocker fixture

### PR DESCRIPTION
**Pull request by fogo**

### CODE REVIEW

File: test_pytest_mock.py [hunk 1]
```diff
@@ -136,3 +136,26 @@ def test_mocker_has_mock_class_as_attribute_for_instantiation():
...
+
+
+def test_mocker_spy(mocker):
+    class Foo(object):
+
+        def bar(self, arg):
+            return arg * 2
+
+    foo = Foo()
+    spy = mocker.spy(foo, 'Bar')
```
Comment (nicoddemus) [inline test_pytest_mock.py]: You mean `'bar'` here, right? :)

**Comment by nicoddemus** [comment]
Could you also please announce on the README about the new `spy` and `stub` methods? Include a simple usage of each one as well, and add a note below each one like *New in version 0.6*.

File: test_pytest_mock.py (hunk 1 above)
Comment (fogo) [inline test_pytest_mock.py]: Ops, that's what happens when you push in a hurry because of a meeting.

**Comment by nicoddemus** [comment]
Thanks! :smile:

**Pull request merged**